stm32/cdcacm.c

```diff
@@ -316,6 +323,12 @@ usb_cinit(void)
 
 #if defined(BOARD_USB_VBUS_SENSE_DISABLED)
 	OTG_FS_GCCFG |= OTG_GCCFG_NOVBUSSENS;
+
```

Comment:
@bkueng Please see line 64 and use the same pattern to document the IP change and test for it.

Reply:
Yes!